docs/install.md

```diff
@@ -3,26 +3,80 @@ id: install
 title: Install
 ---
 
-Install with `yarn`:
+First, install Prettier locally:
 
-```bash
-yarn add prettier --dev --exact
-# or globally
-yarn global add prettier
+```sh
+npm install --save-dev --save-exact prettier
 ```
 
-_We're using `yarn` but you can use `npm` if you like:_
+Then, create an empty config file to let editors and other tooling know you are using Prettier:
 
-```bash
-npm install --save-dev --save-exact prettier
-# or globally
-npm install --global prettier
+```sh
+echo "{}" > .prettierrc.json
+```
+
+Next, create a `.prettierignore` file to let the Prettier CLI and editors know which files to _not_ format:
+
+```sh
+# It’s usually good to start off from your
+# .gitignore or .eslintignore, if you have one:
+cp .gitignore .prettierignore
+# or:
+cp .eslintignore .prettierignore
+# Then edit with your editor of choice:
+vim .prettierignore
 ```
 
-> We recommend pinning an exact version of prettier in your `package.json` as we introduce stylistic changes in patch releases.
+For example, you might want to ignore `build/` and `coverage/`. And if your project isn’t ready to format, say, HTML files yet, add `*.html`.
 
-If you use `npx` to run Prettier, the version should be pinned like this:
+Now, format all files with Prettier:
 
-```bash
-npx prettier@2.0.5 . --write
```

Comment:
I added this part about `npx` when I found out that that's how some people run Prettier on CI without installing it as a dependency (no idea why they do that though). Would be good to keep this remark somewhere.

Reply:
“some people” – who?

Isn’t it better that we promote the “correct way” (adding it to "devDependencies" in package.json)?

Otherwise we’ll have to say something like “if you want to do it the wrong way, here’s a slightly less wrong way to do it”

Reply:
Makes sense. By "some people" I meant those who complained that Prettier broke their CI when 2.0 was released, but now I haven't been able to find those complains. Probably it's indeed not that important.

Reply:
I added the “Note:” part here:

<img width="818" alt="Screenshot 2020-06-18 at 07 15 15" src="https://user-images.githubusercontent.com/2142817/84980895-8f122180-b133-11ea-8c9f-6460bfd12edf.png">

Reply:
Nicely put. The tabs look good too. Might look even better without the top border.

Reply:
Removed borders and reduced padding:

<img width="806" alt="Screenshot 2020-06-27 at 11 43 43" src="https://user-images.githubusercontent.com/2142817/85919454-7d372980-b86b-11ea-80fb-3de19d69d6c7.png">